src/modules/functions.js

```diff
@@ -0,0 +1,95 @@
+// Get the tasks from local storage
+export const getTasks = () => {
+  let tasks;
+
+  if (localStorage.getItem('tasks')) {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  } else {
+    tasks = [];
+  }
+
+  return tasks;
+};
+
+// Add a task to local storage
+export const storeTask = (tasks) => {
```

Comment:
[naming] you could have named the ["storeTask"](https://github.com/Timbar09/To-Do-list/blob/main/src/modules/functions.js#L15) function a more fitting descriptive name like "UpdateTasks".

Reply:
Thanks @Oscah01